ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/107462_footer/112379_footer-links-navigation.cy.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types ='Cypress'/>
+
+Cypress._.each(['ipad-mini'], (viewport) => {
+	describe(`112379: Footer Links on ${viewport}`, () => {
+		beforeEach(() => {
+			cy.login();
+			cy.viewport(viewport);
+	    });
+
+        it('TC01: should navigate to Get Support email link', () => {
+          cy.get('h2').should('contain', 'Get support');
+          cy.get('li').should('contain', 'Email');
+          cy.get('[data-cy="get-support-email"]')
+            .should('contain', 'regionalservices.rg@education.gov.uk');
+        });
+
+        it('TC01: should navigate to Give Feedback link', () => {
+          cy.get('h2').should('contain', 'Give feedback');
+          cy.get('li').should('contain', 'Email');
+          cy.get('#footer-feedback-link')
+            .should('contain', 'Give feedback on Prepare conversions and transfers (opens in a new tab)');
+        });
+
+        it('TC01: should not have unwanted content', () => {
```

Comment:
Is this test offering much value?

Reply:
Not really however there was an acceptance criteria where it was mentioned that this text must not be present. So thought to cover. Also recently we have notice text appearing and missing from Dev env. so covered it :)

Reply:
In my view its best to check these things manually. These tests will exist for the lifetime of the application ever increasing the test run time.  If we add a test for all the things that don't happen or are not there then the tests will take a really long time to run.